Dimas Agung Yudono

dimasday90 opened the pull request:
SECRET=secret
PORT=3000
SALT=10
CLIENT_ID=921344250171-shfc9isl6pb5526l054jlqr2c4h4luju.apps.googleusercontent.com
CLIENT_SECRET=f3CRF4FEiIgXRvsXbWyoDeDX
GPWD=g8528648asA482Jkki

ada kendala di google sign in dan delete

dimasday90 committed:
initial commit

dimasday90 committed:
add client and modified server

dimasday90 committed:
just finish login, register, logout, create, and read all, 3 features left: gSignIn, update, delete

dimasday90 committed:
google sign in dan delete function not done yet

dhimashary commented on client/assets/js/main.js:
```diff
@@ -0,0 +1,343 @@
+        url: `http://localhost:3000/todos/${id}`,
+        headers: {
+            token: localStorage.token
+        }
```
gagal gara2 kurang kurung tutup

dhimashary commented on client/assets/js/main.js:
```diff
@@ -0,0 +1,343 @@
+        method: 'POST',
+        url: 'http://localhost:3000/gLogin',
+        headers: {
+          idToken
```
google sign in gagal gara2 ketika req.headers diterima key nya jadi kecil semua , jadi idtoken di server, coba ya

dhimashary commented:
- belom handle date todo (due date < date now)
- delete masih gagal
- gSignin masih gagal

Jika ada komentar pull req yang kurang jelas segera hubungi buddy

dhimashary commented on server/middlewares/authentication.js:
```diff
@@ -0,0 +1,32 @@
+            where: {
+                id: decoded.id,
+                email: decoded.email,
+                password: decoded.password
```
dilarang simpan password di decoded!

dhimashary commented on server/middlewares/authorization.js:
```diff
@@ -0,0 +1,19 @@
+            UserId: req.decoded.id
+        }
+    })
+        .then(todo => {
```
disini perlu bandingkan todo.userId dengan decodedId !

dimasday90 committed:
1. fixed delete button,
2. adding delete confirm,
3. fixed google sign in features,
4. adding due date validation

dimasday90 commented:
Resolve some bugs:
fixing delete button, add delete confirm feature, fixed google sign in feature, and add due date validation
